src/analysis/ExpressionExplorer.jl

```diff
@@ -806,10 +908,10 @@ explore_module_definition!(expr, scopestate; module_depth::Number = 1) = Symbols
 "Go through a quoted expression and use explore! for :\$ expressions"
 function explore_interpolations!(ex::Expr, scopestate)
     if ex.head == :$
-        explore!(ex.args[1], scopestate)
+        return explore!(ex.args[1], scopestate)::SymbolsState
```

Comment:
curious to why type assertions are sometimes needed :thinking:

Reply:
Not really for TTFX. Here, the compiler just couldn't figure out the return type via type inference which is not good for runtime performance. I don't think that it will affect TTFX (except that a subsequent function call in that function can now be precompiled in the precompilation phase because the compiler knows what for what type to compile...)